packages/palette/src/elements/HorizontalOverflow/HorizontalOverflow.tsx

```diff
@@ -10,7 +10,7 @@ import {
   padding,
   PaddingProps,
 } from "styled-system"
-import { visuallyDisableScrollbar } from "../.."
+import { visuallyDisableScrollbar } from "../../helpers/visuallyDisableScrollbar"
```

Comment:
I think this may be why this component didn't work correctly in Force (!?) — switched to a patch release.